frontend/src/components/student/AllActivities/CombatTask/CombatTask.js

```diff
@@ -91,33 +93,48 @@ export default function CombatTask() {
             </HeaderCol>
             <div>
               <h5>{task.description}</h5>
+              {task.points !== null && (
+                <>
+                  <SmallDivider />
+                  <p>
+                    Zdobyte punkty: <strong>{task.points}</strong>
+                  </p>
+                  {task.remarks && (
+                    <>
+                      <p>Uwagi od prowadzącego:</p>
+                      <p>{task.remarks}</p>
+                      {task.feedbackFile && <FeedbackFileService feedbackFile={task.feedbackFile} />}
+                    </>
+                  )}
+                </>
+              )}
               <SmallDivider />
-              <p>
-                Zdobyte punkty: <strong>50 / 100</strong> {/*//TODO: info from endpoint*/}
-              </p>
-              <p>
-                {task?.content && (
-                  <>
-                    <strong>Uwagi prowadzącego:</strong> <br /> {task.content}
-                  </>
-                )}
-              </p>
-              <SmallDivider />
-              <RemarksCol>
-                <h4>Odpowiedź:</h4>
-                <RemarksTextArea value={answer} onChange={handleAnswerChange} />
-              </RemarksCol>
-
+              {task.points === null && (
+                <RemarksCol>
+                  <h4>Odpowiedź:</h4>
+                  <RemarksTextArea value={answer} onChange={handleAnswerChange} />
+                </RemarksCol>
+              )}
               <FileService
                 task={task}
                 setFile={setFileBlob}
                 setFileName={setFileName}
                 setIsFetching={setIsFetching}
                 isFetching={isFetching}
+                isRevieved={!!task.remarks}
               />
             </div>
-            <SendTaskButton disabled={!fileName && answer === ''} onClick={() => sendAnswer()}>
-              {isFetching ? <Spinner animation={'border'} /> : <span>Wyślij</span>}
+            <SendTaskButton
+              disabled={(!fileName && answer === '') || task.remarks !== null}
+              onClick={() => sendAnswer()}
+            >
+              {isFetching ? (
+                <Spinner animation={'border'} />
+              ) : task.remarks === null ? (
+                <span>Wyślij</span>
+              ) : (
+                <span>Aktywność została oceniona</span>
```

Comment:
skoro po sprawdzeniu aktywności ten button ma taką wartość i jest niedostępny to ja bym wyłączył widoczność inputa do plików - o tego:
![image](https://user-images.githubusercontent.com/63460537/185759529-42d6017e-7638-4b34-9f7c-71250491084c.png)